projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.css

```diff
@@ -10,7 +10,7 @@
 .scroll-container {
   height: 100vh;
   overflow-y: auto;
-  padding: 8px;
+  padding: 8px 8px 8px 12px;
```

Comment:
I think you can revert this change after you fix mat-sidenav-container

Reply:
Yes, you are right.